SurveyApp.DAL/DataContext/ApplicationContext.cs

```diff
@@ -0,0 +1,38 @@
+using SurveyApp.DAL.EntityModels;
+using System.Data.Entity;
+using Microsoft.AspNet.Identity.EntityFramework;
+
+namespace SurveyApp.DAL.DataContext
+{
+    public class ApplicationContext : IdentityDbContext<ApplicationUser>
+    {
+        private static ApplicationContext _instance;
+
+        static ApplicationContext()
+        {
+            Database.SetInitializer(new DropCreateDatabaseIfModelChanges<ApplicationContext>()); 
```

Comment:
Consider using MigrateDatabaseToLatestVersion instead of DropCreateDatabaseIfModelChanges to prevent loss of data after recreating of data base.